### PR TITLE
Feat(ngx datatable)/provide support to disable row

### PR DESCRIPTION
**Pull request by chintankavathia**
**What kind of change does this PR introduce?** (check one with "x")

- [ ] Bugfix
- [x] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Other... Please describe:

**What is the current behavior?** (You can also link to an open issue here)
Today there is no direct way to mark datatable row as disabled
**What is the new behavior?**
This PR will provide option to make row disable 
https://code.siemens.com/simpl/simpl-element/-/issues/630

**Does this PR introduce a breaking change?** (check one with "x")

- [ ] Yes
- [x] No

If this PR contains a breaking change, please describe the impact and migration path for existing applications: ...

**Other information**:

### CODE REVIEW

**Comment by chintankavathia** [comment]
@timowolf , @fh1ch Please review this MR !

Thanks !!

**Review by timowolf — COMMENTED** [review]
Many thanks for this great PR. This will become a great new feature of the table. I only made some comments on naming, style and docs.

File: projects/ngx-datatable/src/lib/themes/_rows.scss [hunk 1]
```diff
@@ -0,0 +1,25 @@
+$disable-row-text-color: #83888E;
+
+.ngx-datatable {
+  .row-disabled {
+    color: $disable-row-text-color;
+
+    .datatable-body-cell-label,
+    input,
+    select {
+      color: $disable-row-text-color;
```
[the remark this answers is not in the document]
Comment (chintankavathia) [reply projects/ngx-datatable/src/lib/themes/_rows.scss:7]: As discussed, removed inputs and select color styles and keeping only label color to be applied in the scenario when user do not use custom component template.

**Review by timowolf — COMMENTED** [review]
Works like a charm and looks nice. Many thanks. Awesome change.

**Review by timowolf — APPROVED** [review]

**Pull request merged**